packages/node/src/plugins/NodeFederationPlugin.ts

```diff
@@ -43,6 +55,13 @@ export const parseRemotes = (remotes: Record<string, any>) =>
     return acc;
   }, {} as Record<string, string>);
 // server template to convert remote into promise new promise and use require.loadChunk to load the chunk
+/**
+ * This function generates a remote template.
+ * @function
+ * @param {string} url - The url of the remote
+ * @param {any} global - The global variable
+ * @returns {string} The generated remote template
+ */
 export const generateRemoteTemplate = (
```

Comment:
The function 'parseRemotes' does not handle the case when the remotes object is null or undefined. This could lead to unexpected behavior or errors. Add a check to handle such cases.